<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComponentsDB</class>
 <widget class="QWidget" name="ComponentsDB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1234</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="minimumSize">
      <size>
       <width>192</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>192</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>-1</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QTreeWidget
{
    background-color: #9DC6E6;
    font: 14px &quot;黑体&quot;;
}

QHeaderView::section {
    background-color: #9DC6E6; /* 设置背景颜色 */
    color: black; /* 设置文字颜色 */
    border: none;
    font: 14px &quot;黑体&quot;;
}</string>
     </property>
     <property name="headerHidden">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>数据库</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="page_fan"/>
     <widget class="QWidget" name="page_fanCoil"/>
     <widget class="QWidget" name="page_air_noi_single_fan"/>
     <widget class="QWidget" name="page_air_noi_double_fan"/>
     <widget class="QWidget" name="page_VAV_terminal"/>
     <widget class="QWidget" name="page_circular_damper"/>
     <widget class="QWidget" name="page_rect_damper"/>
     <widget class="QWidget" name="page_air_diff"/>
     <widget class="QWidget" name="page_pump_send"/>
     <widget class="QWidget" name="page_staticBox_grille"/>
     <widget class="QWidget" name="page_disp_vent_terminal"/>
     <widget class="QWidget" name="page_other_send_terminal"/>
     <widget class="QWidget" name="page_static_box"/>
     <widget class="QWidget" name="page_duct_with_multi_ranc"/>
     <widget class="QWidget" name="page_tee"/>
     <widget class="QWidget" name="page_pipe"/>
     <widget class="QWidget" name="page_elbow"/>
     <widget class="QWidget" name="page_reducer"/>
     <widget class="QWidget" name="page_silencer"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
